version/version.go

```diff
@@ -1,7 +1,7 @@
 package version
 // AUTO-GENEREATED. DO NOT EDIT
-// 2015-08-14 09:56:50.742727493 -0400 EDT
+// 2016-01-30 10:22:43.857076493 +0100 CET
 
-// VERSION is the generated version from /home/vbatts/src/vb/tar-split/version
-var VERSION = "v0.9.6-1-gc76e420"
+// VERSION is the generated version from /home/runcom/src/github.com/vbatts/tar-split/version
+var VERSION = "v0.9.11-1-gd50e5c9"
```

Comment:
I know it's wrong I just regenerated this to remind you it needs to be updated :)